service/internal/integrationtest/s3/checksum_test.go

```diff
@@ -541,6 +541,38 @@ func TestInteg_RequireChecksum(t *testing.T) {
 	}
 }
 
+func TestInteg_RequireChecksumWithoutRequestAlgorithmMember(t *testing.T) {
+	params := &s3.PutBucketOwnershipControlsInput{
+		Bucket: &setupMetadata.Buckets.Source.Name,
+		OwnershipControls: &types.OwnershipControls{
+			Rules: []types.OwnershipControlsRule{
+				{
+					ObjectOwnership: types.ObjectOwnershipBucketOwnerPreferred,
+				},
+			},
+		},
+	}
+
+	t.Logf("putting bucket ownership control: %q", *params.Bucket)
+	result, err := s3client.PutBucketOwnershipControls(context.Background(), params)
+	if err != nil {
+		t.Fatalf("expect no error, got %v", err)
+	}
+	computedChecksums, ok := s3.GetComputedInputChecksumsMetadata(result.ResultMetadata)
+	if !ok {
+		t.Fatalf("expect computed checksums metadata present, got %q", computedChecksums)
```

Comment:
```suggestion
		t.Fatalf("expect computed checksums metadata present, got %q", result)
```

Sorry if I wasn't clear with my suggestion. If we print `computedChecksums` we already know the result is nil, so we don't get anything new by printing the value. If we print the whole result, we can see what we got and hopefully have a better clue of what went wrong